fastlane/metadata/android/release_notes.xml

```diff
@@ -1,85 +1,8 @@
-<ar>
```

Comment:
There's a lot of red in this file. I decided to submit without localized release notes to prioritize shipping the signup fix.

Reply:
Translations will be back tomorrow. It should be possible to update the metadata even when the binary is approved, so it's probably worth it to check again tomorrow and add the missing translations.